src/Ng/Jobs/LoopingNgJob.cs

```diff
@@ -38,8 +38,15 @@ protected LoopingNgJob(ILoggerFactory loggerFactory)
                 }
 
                 Logger.LogInformation("Running job.");
-                await RunInternal(cancellationToken);
-                Logger.LogInformation("Job finished!");
+                try
+                {
+                    await RunInternal(cancellationToken);
+                    Logger.LogInformation("Job finished!");
+                }
+                catch (TransientException te)
+                {
+                    Logger.LogWarning("Job execution ended due to a transient exception: {0}. {1}", te.Message, te.InnerException?.StackTrace??te.StackTrace);
```

Comment:
The idea is interesting. The failure could be due to low resources on the machine, waiting could be beneficial in case like this. I would let the job with the regular time interval for now.